Remotion/Web/Development.Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,6 @@
+### New Rules
```

Comment:
@a-ctor  & @Frazerus There was a discussion about not needed this if I remember correctly. Apparently, this semantic has now changed. Thoughts?

Reply:
We did not need it because compile did not complain back then. Now it does so we should add them as is normal with all analyzer projects. To this day, I don't know if they have any technical relevance or are just here for documentation purposes...